test/kb_SPAdes_server_test.py

```diff
@@ -348,6 +348,11 @@ def test_single_reads(self):
         self.run_success(
             ['single_end'], 'single_out',
             dna_source='None')
+
+    def test_single_reads_with_mcl(self):
+        self.run_success(
+            ['single_end'], 'single_out',
+            min_contig_length=1000, dna_source='None', contig_count=6)
```

Comment:
Needed to add this test to exercise the changes in the *Impl.py file, that block was previously uncovered